README.md

```diff
@@ -52,22 +45,28 @@ Go to your SonarQube instance administration console and open Update Center. Fin
 Use of the plugin does not differ much from regular SonarQube analysis. However, as rules are often tied to a certain AEM version and its components (Felix, Sling), we've introduced the `aemVersion` analysis property.
 
 Each rule defines supported AEM version or version range. Most of the rules are universal.
-By providing the AEM version parameter, you can instruct the Sonar Runner to only use only a subset of rules applicable to a particular AEM version. When the parameter is not provided then a default AEM version is used (currently 6.4)
+
+Rules are not available in [<img src="https://rules.sonarsource.com/images/logos/SonarLint-black.svg" height="28" alt="Not available in SonarLint">](https://www.sonarlint.org/)
```

Comment:
Are we going ahead with a version incompatible with SonarLint then? I think this could be a common use case.